docs/DeveloperGuide.adoc

```diff
@@ -356,6 +356,56 @@ For example, `a` to add a piece of text, arrow keys to adjust the position of th
 
 // end::templates[]
 
+// tag::stats[]
+=== Statistics feature 1: Like Meme
+
+Unlike other file management system, people have preference of some memes over others.
```

Comment:
The part on file management system doesnt really make sense

Reply:
Huh? Ppl don't just mark some files "Prefered" over others i guess. It is different from tag as well.